fairroot/base/LinkDef.h

```diff
@@ -35,7 +35,7 @@
 #pragma link C++ class FairParticle+;
 #pragma link C++ class FairPrimaryGenerator+;
 #pragma link C++ class FairRecoEventHeader+;
-#pragma link C++ class FairRootManager+;
+#pragma link C++ class FairRootManager;
```

Comment:
If I am correct this will still create the streamer but an older and not IO optimised form. To not create the streamer a `-` shold be added to the class name. Unfortunately I can't find the proper documentation. I only found 
https://root.cern/manual/io_custom_classes/
which doesn't describe the feature.
I will try to do some tests to clarify this.

Reply:
> Note
> 
> The `+` after the class name enables extra features and performance improvements in the I/O of the type.

Okay, that sounds like what Florian is looking at.

> Remember to always add it to your linkdef directives.

Reading that I would think, that we should just keep it for the time being.

And as I understand it, the `0` version in the ClassDefOverride already disables the streamer.

Reply:
hm, the streamer for the TObject base class remains. It looks to me as if one cannot use `-`-postfix on classes that inherit from `TObject`?

Reply:
`ClassDefOverride(FairRootManager, 0)` +

1. `#pragma link C++ class FairRootManager-;`
   ```
   /usr/bin/ld: fairroot/base/CMakeFiles/Base.dir/steer/FairRootManager.cxx.o:(.data.rel.ro._ZTV15FairRootManager[_ZTV15FairRootManager]+0x1e0): undefined reference to 'FairRootManager::Streamer(TBuffer&)'
   /usr/bin/ld: fairroot/base/CMakeFiles/Base.dir/G__Base.cxx.o: in function 'ROOT::streamer_FairRootManager(TBuffer&, void*)':
   /home/dklein/projects/FairRoot3/build/fairroot/base/G__Base.cxx:5486:(.text+0x171bc): undefined reference to `FairRootManager::Streamer(TBuffer&)'
   ```

2. `#pragma link C++ class FairRootManager;`
   
   ```cpp
   void FairRootManager::Streamer(TBuffer &R__b)
   {
      // Stream an object of class FairRootManager.
   
      TObject::Streamer(R__b);
   }
   ```

3. `#pragma link C++ class FairRootManager+;`

   ```cpp
   void FairRootManager::Streamer(TBuffer &R__b)
   {
      // Stream an object of class FairRootManager.
   
      if (R__b.IsReading()) {
         R__b.ReadClassBuffer(FairRootManager::Class(),this);
      } else {
         R__b.WriteClassBuffer(FairRootManager::Class(),this);
      }
   }
   ```

Reply:
@fuhlig1 Let me know, if I should put the `+` back.